src/Stratis.Bitcoin.Features.Miner.Tests/PosBlockAssemblerTest.cs

```diff
@@ -440,23 +442,20 @@ private void SetupConsensusLoop()
                 {
                     this.callbackRuleContext = c;
                 }).Verifiable();
-
-            this.consensusLoop.Setup(c => c.Validator)
-                .Returns(this.validator.Object);
         }
 
-        private void SetupValidator()
+        private void SetupRulesEngine(ConcurrentChain chain)
```

Comment:
What happened to the indentation here?

Reply:
It lost its way somehow :) Fixed in next commit